packages/pwa-kit-extension-sdk/src/shared/utils/helpers.ts

```diff
@@ -75,22 +71,28 @@ const isApplicationExtensionEntryArray = (entry: ApplicationExtensionEntryArray)
 
 /**
  * Normalize and expand the extension configuration array so that it is easier to process.
- * @param {{String, Object}[]} extensions - The extensions configuration value as defined in the PWA-Kit config.
- * @returns {Object[]} extensions - The extensions array in object form.
+ * @param extensions - The extensions configuration value as defined in the PWA-Kit config.
+ * @returns a list of extensions all in tuple format
  *
  * @example
- * const result = expand(["store-finder", ["account-pages", {singlePage: true}], './extensions/local-extension']);
+ * const result = expand([
+ *   '@salesforce/extension-a',
+ *   ['@salesforce/extension-b', {foo: 'bar'}],
+ *   ['@salesforce/extension-c']
+ * ]);
  * console.log(result)
- * // [["@salesforce/extension-store-finder", {}], ["@salesforce/extension-account-pages", {singlePage: true}], ["/home/project/extensions/local-extension", {}]]
+ * [
+ *   ['@salesforce/extension-a', {enabled: true}],
+ *   ['@salesforce/extension-b', {enabled: true, foo: 'bar'}],
+ *   ['@salesforce/extension-c', {enabled: true}]
+ * ]
  */
-export const expand = (
-    extensions: ApplicationExtensionEntry[] = []
-): ApplicationExtensionEntryArray[] =>
+export const expand = (extensions: unknown[] = []): ApplicationExtensionEntryArray[] =>
     extensions
         .filter((extension) => Boolean(extension))
         .map((extension) => {
             const thing: [string, any] = Array.isArray(extension)
```

Comment:
nit: The variable name `thing` is not very descriptive. Can we give it a better name like `extensionEntry`?